### DISCUSSION TITLE
Is it possible to have a bit of guidance?

### DISCUSSION THREAD

**Discussion by ShaysFrame** [hawkkiller/sizzle_starter#379] [General]
How do I learn about this template in detail? How do we extend new features? are there any video tutorials or any guidance on how to build from the template to a fully functional app?

**Comment by hawkkiller** [comment]
Currently there is no up-to-date documentation and you can learn its contents on your own.

It is not big, but provides essential things for any app. See the initialization, folder and project structure.

It also has built in things like error tracking, logger, analytics.

**Comment by ShaysFrame** [comment]
Do you have like a demo app that has at least one or two features like
authentication, bottom navigation and some demo page that uses this
structure?

I’m learning flutter so a bit lost seems like there’s nothing i know from
this code base of yours lol 😂.

**Reply by hawkkiller** [reply]
Alright 🥹

I have recently created a project called Pokemon, also have catty repository, check them out.

This template is not easy to tackle if you're starting.

I'd recommend to learn the basics from flutter docs (there's a lot of useful stuff). And when you're back, hopefully I will write more docs.
